packages/url_launcher/url_launcher/README.md

```diff
@@ -53,6 +53,28 @@ Common schemes supported by both iOS and Android:
 
 More details can be found here for [iOS](https://developer.apple.com/library/content/featuredarticles/iPhoneURLScheme_Reference/Introduction/Introduction.html) and [Android](https://developer.android.com/guide/components/intents-common.html)
 
+### Encoding URLs
+
+URLs should be safely encoded, espeically when including spaces or other special characters. We can do this using dart's `Uri` helper methods included in `dart:core`:
+```dart
+import 'dart:core';
+import 'package:url_launcher/url_launcher.dart';
+
+final Uri _emailLaunchUri = Uri(
+  scheme: 'mailto',
+  path: 'smith@example.com',
+  queryParameters: {
+    'subject': 'Example Subject & Symbols are allowed!'
+  }
+);
+
+final String _emailLaunchString = _emailLaunchUri.toString();
+
+// ...
+
+launch(_emailLaunchString);
```

Comment:
Maybe collapse this last section to just:
```
launch(emailLaunchUri.toString());
```
Extracting it to a variable if someone wants to do other things with the string first is something they can do trivially, so having shorter example code seems like the better option.